documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -417,6 +417,16 @@
                "hz":0.0,
                "counter":0,
                "roughness":0.0
+            },
+            "location_requests":{ // measures number of incoming key server location requests
```

Comment:
This comment says it measures incoming requests, but the computation appears to be based on answered requests